synapse/lib/layer.py

```diff
@@ -1014,10 +1014,10 @@ async def setLayerInfo(self, name, valu):
         return valu
 
     async def stat(self):
-        return {
-            'nodeeditlog_indx': (self.nodeeditlog.index(), 0, 0),
-            **self.layrslab.statinfo()
-        }
+        ret = {**self.layrslab.statinfo()}
+        if hasattr(self, 'nodeeditlog'):
```

Comment:
This is a code smell. Perhaps we should just initialize nodeeditlog to None at anit time.